docs/source/index.rst

```diff
@@ -34,17 +35,26 @@ System Requirements
 What To Do Next
 ===============
 
-Advanced users who have already installed Python can proceed to the installation instructions.
-Less experienced users should start with the basic setup instructions, which will introduce some
-basics for working with the terminal and installing the dependencies to run this package.
+Advanced users who have already installed Python
+can proceed to instructions for **Installing submitr**.
+
+Less experienced users should start with instructions
+for **Installing Prerequisites**,
+which will introduce some basics for working with the terminal
+and installing the dependencies to run this package.
 
+Although at some point **submitr** might offer the ability to
```

Comment:
Update from submitr to SubmitCGAP:
```suggestion
can proceed to instructions for **Installing SubmitCGAP**.

Less experienced users should start with instructions
for **Installing Prerequisites**,
which will introduce some basics for working with the terminal
and installing the dependencies to run this package.

Although at some point **SubmitCGAP** might offer the ability to
```

Reply:
I fixed the two stray references. Thanks for noting the stylistic thing. I'd seen that, too. I made those changes to bring diffing into best alignment between the repos, but I'm not sure why I started down that route. It might be because `submitr` looks better in some other font, lacking all the uppercase. I'm not sure that can be fully reconciled but I'm less worried about it for now. Maybe another day.